### PR TITLE
Disabled pinch zooming with two fingers.

### PR DESCRIPTION
**Pull request by JeffersonMat**
I've added a line to the header meta tag, to disable user scalability. 
And also, added touch-action to CSS to only enable x and y action for scrolling.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -190,6 +190,7 @@ html,
 body {
 	height: 100%;
 	overflow: hidden;
+    touch-action: pan-x pan-y;
```
Comment (colbr) [inline src/styles/main.scss:193]: Make sure to fix indentation. I'm not sure how this wasn't caught by Prettier though, can you double check Prettier is running on save?

**Review by colbr — APPROVED** [review]
Nice @JeffersonMat 

Requested a fix for some indentation, but I won't let that block the merge. Push it through with proper indentation then merge :)

File: src/styles/main.scss (hunk 1 above)
Comment (JeffersonMat) [reply src/styles/main.scss]: Done. I have run npm, however, I don't seem to see prettier running on commits. I'll dig into this to find out what's going on.

**Comment by JeffersonMat** [comment]
The indentation issue has been fixed.

**Pull request merged**